@kiva/kv-components/vue/KvWwwHeader/KvHeaderMyKivaMenu.vue

```diff
@@ -6,7 +6,7 @@
 		>
 			<span>Portfolio</span>
 			<span class="tw-text-secondary">|</span>
-			<span class="tw-text-eco-green-3">{{ balance }}</span>
+			<span class="tw-text-eco-green-3">${{ balance }}</span>
```

Comment:
Do we want to just add a `$` or use `numeral` to format? I'm not sure what we did in the previous header.